libs/qCC_db/src/ccPointCloud.cpp

```diff
@@ -6120,3 +6120,119 @@ bool ccPointCloud::shiftPointsAlongNormals(PointCoordinateType shift)
 
 	return true;
 }
+
+ccPointCloud::Grid::Grid()
+	: w(0)
+	, h(0)
+	, validCount(0)
+	, minValidIndex(0)
+	, maxValidIndex(0)
+{
+	sensorPosition.toIdentity();
+}
+
+ccPointCloud::Grid::Grid(const Grid& grid)
```

Comment:
Wouldn't having `Grid(const Grid& grid) = default` in the `Grid` class declarion be shorter ?

Reply:
That would probably work indeed.